test/integration/roles/test_amazon_rds/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# create and kill an RDS
+
+# Improvements expected in later versions of this test case
+
+# use present instead of create
+# use absent instead of delete
+# use SSD
+
+
+# Start with creation - N.B. if the RDS already exists from a previous run this
+# may not change anything so we don't insist on that
+
+  - name: create RDS
+    tags: test_amazon_rds
+    local_action:
+      module: rds
+      region: "{{ ec2_region }}"
+      command: create
+      instance_name: "{{ resource_prefix }}-rds"
+      db_engine: postgres
+      size: 10
+      instance_type: "db.t2.micro"
+      username: "{{ rds_master_db_user }}"
+      password: "{{ rds_db_master_password }}"
+      publicly_accessible: no
+      wait: yes
+      wait_timeout: 1200 # 600 doesn't cut it!!
+    environment:
+      EC2_REGION: '{{ec2_region}}'
```

Comment:
Actually; it's a good question;  do you have a clear idea / statement what the *right* way to do this is.  By "right" I mean I want the following

- the test cases should use the credentials in credentials.yml and *never* any credentials from the environment or other configuration of the user's account
- there shouldn't be duplication (see also https://github.com/ansible/ansible/pull/22499 - especially comments from @s-hertel ) so it should be global
- it should be easy to override for tests which want to use a different config (e.g. _test environment variable EC2_REGION_ in ec2_key.py) 
- preferably the method is used elsewhere and/or can be used many places in the cloud tests

thanks in advance for any help you can give - Michael